### ISSUE TITLE
Update Telemetry Client

### ISSUE THREAD

**Issue by brandonh-msft** [dotnet/upgrade-assistant#1207]
The DevDiv Data Platform team is in the process of modernizing and simplifying our offering.  As part of this we’re taking an inventory of all the products we support to see if continued support is required.  As part of continued support we’ll be asking product owners to update their telemetry clients in the coming semester as older telemetry clients are no longer supported which means we cannot make any data delivery guarantees until you upgrade.  
Here are the decisions that we’ll looking for from you by 7/22 
•	Do you need continued support for your data pipeline?
•	Will you be able to update your telemetry client in the next semester in January with us?  We can help with the decision process of which telemetry client to upgrade to.

**Comment by sunandabalu** [comment]
What telemetry client is being referred to here? UA collects telemetry data using the inbuilt .NET CLI telemetry as part of .NET SDK.

**Comment by brandonh-msft** [comment]
Yup. From what I'm interpreting, it needs an update. The link in the msg didn't come thru: https://docs.microsoft.com/en-us/azure/azure-monitor/app/sdk-support-guidance
